spec/helpers/backend.js

```diff
@@ -26,7 +26,7 @@ function backendRequest(params, callback) {
     }
 
     // how many times it will try to request before throwing error
-    var ttl = 3;
+    let ttl = retry && 3 || 0;
```

Comment:
This is too fancy, please use the ternary:
```js
let ttl = retry ? 3 : 0;
```
Or just delete this variable and declare the function as:
```js
function backendRequest(params, callback, ttl = 0) {
```
Then when calling it, instead of true or false just use the ttl value

Reply:
thanks, I'll change to your second version with `ttl = 0`

Reply:
```
/arch/naspeh/works/sourcefabric/superdesk-dev/client-core/spec/helpers/backend.js
  45:13  error  Assignment to function parameter 'ttl'  no-param-reassign

✖ 1 problem (1 error, 0 warnings)
```
because below we have
```
ttl -= 1;
```
So I'll use different parameter name then

Reply:
Done

Reply:
You're right, I didn't see that initially, LGTM! Thanks